self_hosting_machinery/inference/inference_hf.py

```diff
@@ -149,38 +149,32 @@ def __init__(self,
 
         self._device = "cuda:0"
         token = huggingface_hub_token()
-        for local_files_only in [True, False]:
```

Comment:
what's the reason of this change?
how we should handle the case when we have no connection to internet?

Reply:
It's handled automatically, there is no need for this code